packages/cosmos_ui_components/lib/components/cosmos_image_button.dart

```diff
@@ -1,30 +1,27 @@
-import 'package:cosmos_ui_components/cosmos_ui_components.dart';
 import 'package:flutter/material.dart';
 
 class CosmosImageButton extends StatelessWidget {
-  final String assetUrl;
-  final VoidCallback onPressed;
+  final String asset;
+  final double height;
+  final double width;
+  final VoidCallback? onTap;
 
   const CosmosImageButton({
     Key? key,
-    required this.assetUrl,
-    required this.onPressed,
+    required this.asset,
+    required this.onTap,
```

Comment:
its nullable field, we can remove the `required` keyword here

Reply:
Updated